resources/testdata/Cargo.toml

```diff
@@ -97,6 +97,7 @@ cldr_json_glob = [
     "cldr-core/supplemental/ordinals.json",
     "cldr-core/supplemental/plurals.json",
     "cldr-dates-full/main/$LOCALES/ca-gregorian.json",
+    "cldr-numbers-full/main/$LOCALES/numbers.json",
```

Comment:
The "random keys" are under the heading

```
# icu4x_testdata metadata: schema defined in ./src/metadata.rs
[package.metadata.icu4x_testdata]
```

I added this mechanism for providing configurations to the testdata tool fairly early on.  I thought it was nicer to use Cargo.toml's "package.metadata" rather than, e.g., making a new TOML file.

Reply:
Ah, I should get better at spotting nesting. I see that it's `package.metadata`. Thanks for the clarification.